app/src/common/shared/org/mozilla/vrbrowser/utils/InternalPages.java

```diff
@@ -134,9 +136,10 @@ public static String createErrorPageDataURI(Context context,
                 showSSLAdvanced = false;
         }
 
-        html = html
-                .replace("%url%", uri)
-                .replace("%advancedSSLStyle%", showSSLAdvanced ? "block" : "none");
+        if (uri != null) {
+            html = html.replace("%url%", uri);
+        }
```

Comment:
I wonder if it would be better to do:
`html = html.replace("%url%", uri != null ? uri : "NULL");`